src/selectors/__tests__/exportContext.ts

```diff
@@ -0,0 +1,78 @@
+import { initialState, reducerFlow } from '../../util'
+import { newThought, setCursor, importText } from '../../reducers'
+import { HOME_PATH } from '../../constants'
+import exportContext from '../exportContext'
+
+describe('exportContext', () => {
+  it('test when meta and archived thoughts are included', () => {
```

Comment:
This test doesn't do what it says: it can't test when meta and archived thoughts are included if there are no meta or archived thoughts. Either you mean to test the base case of `exportContext` with no arguments and no meta thoughts, or you mean to test the case where meta/archived thoughts are included.

Reply:
Updated.